templates/silicon/index.html

```diff
@@ -0,0 +1,550 @@
+            hi_def=True,
+            loading="auto",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/4bbeaccd-risc-v-logo.svg",
+            alt="Risc-V logo",
+            width="292",
+            height="290",
+            hi_def=True,
+            loading="auto",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/5e1b56d3-si-five-logo.svg",
+            alt="SiFive logo",
+            width="292",
+            height="290",
+            hi_def=True,
+            loading="auto",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/c5d6e185-allwinner-logo.png",
+            alt="Allwinner logo",
+            width="292",
+            height="290",
+            hi_def=True,
+            loading="auto",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/8c8f672b-star-five-logo.svg",
+            alt="StarFive logo",
+            width="292",
+            height="290",
+            hi_def=True,
+            loading="auto",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/fd56b5a4-qualcom-logo.svg",
+            alt="Qualcom logo",
+            width="292",
+            height="290",
+            hi_def=True,
+            loading="auto",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+      </div>
+      <hr/>
+      <p class="u-float-right">
+        <a href="https://canonical.com/partners/find-a-partner?filters=ihv-oem">
+          See all silicon partners
+        </a>
+      </p>
+    </div>
+  </div>
+</section>
+<section class="p-strip">
+  <div class="row">
+    <hr />
+    <div class="col-6">
+      <div class="row">
+        <div class="col-5">
+          <h2>Benefits of the Silicon Partner Programme</h2>
+        </div>
+      </div> 
+    </div>
+    <div class="col-6">
+      <ul class="p-list--ticked">
+        <li class="p-list__item u-sv2">
+          <p class="u-no-margin--bottom usv-2">
+            Certified HW and reliable Long Term Support
+          </p>
+        </li>
+        <li class="p-list__item u-sv2">
+          <p class="u-no-margin--bottom usv-2">
+            Custom feature(SDK, toolkits) additions and optimizations
```

Comment:
```suggestion
            Custom feature(SDK, toolkits) additions and optimisations
```
Uk spelling

Reply:
I didn't even notice that. It came directly from the copy docs. 

I'll definitely fix that, seems there are a couple of them in around the page